pkgs/applications/video/glaxnimate/default.nix

```diff
@@ -65,6 +66,11 @@ stdenv.mkDerivation rec {
     python3WithLibs
   ];
 
+  # Translation needs to be separately compiled
+  # https://gitlab.com/mattbas/glaxnimate/-/issues/648
+  postBuild = ''
+    make translations
```

Comment:
Perhaps you could do this using `buildFlags`, so it's included in the same make command?  That way, all the right flags would be applied.  Otherwise, the `makeFlags` and `makeFlagsArray` variables would need to be used here.

Reply:
Thanks, it's done.